webui/react/src/table/columns.tsx

```diff
@@ -0,0 +1,72 @@
+import { ColumnType } from 'antd/lib/table';
```

Comment:
suggestion: how about `src/utils/table.tsx`?

Reply:
chatted about it. will leave it here until we find a better place :)